src/player/camera/instruction/CameraInstruction.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\player\camera\instruction;
+
+use pocketmine\nbt\tag\CompoundTag;
+
+/**
+ * Interface implemented by camera instructions.
+ */
+interface CameraInstruction{
+
+	/**
+	 * Writes this instriction in a CompoundTag in order to be sent to the client.
+	 */
+	public function writeInstructionData(CompoundTag $tag) : CompoundTag;
```

Comment:
Why does this return CompoundTag and the only thing used is the $tag parameter?

Reply:
It's leftovers from an API redesign that I made, this was to make it fluent, I forgot to remove it.